test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -1161,6 +1200,51 @@ func RunBenchmarkPerfScheduling(b *testing.B, outOfTreePluginRegistry frameworkr
 	}
 }
 
+var testSchedulingLabelFilter = flag.String("test-scheduling-label-filter", "integration-test,-performance", "comma-separated list of labels which a testcase must have (no prefix or +) or must not have (-), used by TestScheduling")
```

Comment:
This has the side-effect that importing the package adds the command line flag. Avoiding that side-effect was the reason why it was in the `_test.go` before. This probably doesn't matter, but perhaps we can avoid it by moving all flag registrations into a common `MainTest` function which registers the flags, parses the command line arguments, initializes logging, and the runs tests?

Reply:
Okay, I moved the flags registration to InitTests function that is called in each TestMain